# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/moz.build
# Commit: e802face8e04
# Full Hash: e802face8e042d03933c1495d36fd02a9def7dd9
# Author: Brian Hackett <bhackett1024@gmail.com>
# Date: 2019-07-25 21:51:57
# Regressor Bug: 1554524
# File Overlap Count: 1
# Description:
#   Bug 1554524 Part 4 - Add Debugger interface for instrumenting scripts, r=jimb,tcampbell.
# ==============================================================================

diff -r fc204daba6f3 -r e802face8e04 js/src/moz.build
--- a/js/src/moz.build	Mon Jul 08 09:39:52 2019 -1000
+++ b/js/src/moz.build	Mon Jul 08 09:41:21 2019 -1000
@@ -292,6 +292,7 @@
     'vm/HelperThreads.cpp',
     'vm/Id.cpp',
     'vm/Initialization.cpp',
+    'vm/Instrumentation.cpp',
     'vm/Iteration.cpp',
     'vm/JSAtom.cpp',
     'vm/JSContext.cpp',